src/util/intOps.ml

```diff
@@ -51,7 +51,7 @@ sig
   val to_int : t -> int
   val of_int64 : int64 -> t
   val to_int64 : t -> int64
-  val of_string : string -> t
+  val of_string : string -> t (* TODO: unused *)
```

Comment:
I think we can keep the `of_string`, it may be handy sometimes.

Reply:
Possibly yes. I guess this was used when we still parsed the strings from CIL constants for big values.